google/detectors/rce/CVE-2017-5638/README.md

```diff
@@ -0,0 +1,14 @@
+# Apache Struts Command Injection via Content-Type Header (CVE-2017-5638) Detector
+
+This checks if an Apache Struts application is vulnerable to Command Injection 
```

Comment:
nit: Text files may not contain trailing whitespace

Reply:
Done